test/ered_SUITE.erl

```diff
@@ -138,7 +148,7 @@ end_per_suite(_Config) ->
 stop_containers() ->
     %% Stop containers, redis-30007 used in t_new_cluster_master
     cmd_log([io_lib:format("docker stop redis-~p; docker rm redis-~p;", [P, P])
-             || P <- ?PORTS ++ [30007]]).
+             || P <- ?PORTS ++ [30007, cluster]]).
```

Comment:
Ah, the container used in create_cluster/0 can get stuck or take a while.. probably good to include it in the stop. Maybe add a comment.